features/gba-dev/europees-kiesrecht-gba.feature

```diff
@@ -0,0 +1,62 @@
+# language: nl
+
+Functionaliteit: Europees kiesrecht
+
+  Scenario: Geen Europees kiesrecht opgegevens, geen Europees kiesrecht (31.10) en geen einddatum uitsluiting Europees kiesrecht (31.30)
+    Gegeven de persoon met burgerservicenummer '000000103' heeft de volgende gegevens
+    | geslachtsnaam (02.40) |
+    | Janssen               |
+    Als gba personen wordt gezocht met de volgende parameters
+    | naam                | waarde                          |
+    | type                | RaadpleegMetBurgerservicenummer |
+    | burgerservicenummer | 000000103                       |
+    | fields              | europeesKiesrecht            |
+    Dan heeft de response een persoon zonder gegevens
+
+  Abstract Scenario: Europees kiesrecht zonder einddatum : <omschrijving>
+    Gegeven de persoon met burgerservicenummer '000000115' heeft de volgende 'inschrijving' gegevens
+    | Europees kiesrecht (31.10) |
```

Comment:
Zou eigenlijk 'europees kiesrecht (31.10)' moeten zijn. We hanteren eigenlijk nergens hoofdletters.

Reply:
aangepast